Aptoide/aptoide-client-v8
# [APP-834] Integrate amplitude and indicative

**What does this PR do?**

This PR aims at implementing Amplitude and Indicative.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Open the app, do the major flows and check by charles and logs that the requests are being correctly sent.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-834](https://aptoide.atlassian.net/browse/APP-834)
[APP-835](https://aptoide.atlassian.net/browse/APP-835)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
